compiler/test/dotty/tools/dotc/CompilerTest.scala

```diff
@@ -149,7 +149,8 @@ abstract class CompilerTest {
 
   private def expectedErrors(filePath: String): List[ErrorsInFile] = expectedErrors(List(filePath))
 
-  private def isNegTest(testPath: String) = testPath.contains("/neg/")
+  private def isNegTest(testPath: String) =
+    testPath.contains("/neg/") || testPath.contains("/neg-custom-args/") || testPath.contains("/neg-tailcall/")
```

Comment:
`testPath.startsWith("/neg")` ?

Reply:
I believe the full path is given here. Something like `../tests/neg/Foo.scala`. I wouldn't bother to do something smarter since this is for the legacy tests

Reply:
ok.